CONTRIBUTING.md

```diff
@@ -448,6 +449,27 @@ _(comments added for clarity)_
 
 &nbsp;
 
+### :convenience_store: &nbsp; Add missing brands
+
+If it exists, we want to know about it!
+
+Some brands aren't mapped enough (50+ times) to automatically be added to the index so this
+is a valuable way to get ahead of incorrect tagging.
+
+1. Before adding a new brand, the minimum information you should know is the correct tagging required for instances of the brand (`name`, `brand` and what it is - e.g. `shop=food`). Ideally you also have `brand:wikidata` and `brand:wikipedia` tags for the brand and any other appropriate tags - e.g. `cuisine`.
```

Comment:
If there isn’t a Wikidata item for the brand yet, we could suggest [creating one](https://www.wikidata.org/wiki/Special:NewItem) and maybe recommend some common values to set the [instance of](https://www.wikidata.org/wiki/Property:P31) property to. Increasingly people will want to add entries for brands that don’t warrant Wikipedia articles, but they can still have Wikidata items.